src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -7252,428 +7267,422 @@ private void makeCoilRecipes() {
-                    300,
-                    30);
-            // Nether
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "netherrack", 64L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "soul_sand", 64L, 0),
-                            GT_ModHandler.getModItem(BiomesOPlenty.ID, "ash", 64L, 0),
-                            GT_Utility.getIntegratedCircuit(3) },
-                    Materials.Lava.getFluid(10000L),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_Ne", 1L, 0),
-                    300,
-                    30);
-            // End
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "end_stone", 64L, 0),
-                            GT_ModHandler.getModItem(Minecraft.ID, "obsidian", 64L, 0),
-                            GT_ModHandler.getModItem(Natura.ID, "Cloud", 64L, 1), GT_Utility.getIntegratedCircuit(4) },
-                    FluidRegistry.getFluidStack("ender", 10000),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_ED", 1L, 0),
-                    300,
-                    30);
-            // Far End Asteroids
-            GT_Values.RA.addAssemblerRecipe(
-                    new ItemStack[] { GT_ModHandler.getModItem(Botania.ID, "tinyPlanetBlock", 1L, 0),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 0),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 1),
-                            GT_ModHandler.getModItem(HardcoreEnderExpansion.ID, "end_stone_terrain", 64L, 2),
-                            GT_Utility.getIntegratedCircuit(5) },
-                    FluidRegistry.getFluidStack("endergoo", 10000),
-                    GT_ModHandler.getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_EA", 1L, 0),
-                    300,
-                    30);
+        // T0 Planets
+        // Overworld
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        getModItem(Botania.ID, "tinyPlanetBlock", 1, 0),
+                        getModItem(Minecraft.ID, "stone", 64, 0),
+                        getModItem(Minecraft.ID, "grass", 64, 0),
+                        getModItem(Minecraft.ID, "sapling", 1, 0),
+                        GT_Utility.getIntegratedCircuit(1))
+                .fluidInputs(Materials.Water.getFluid(10000))
+                .itemOutputs(getModItem(NEIOrePlugin.ID, "blockDimensionDisplay_Ow", 1, 0)).noFluidOutputs()
+                .duration(300).eut(TierEU.RECIPE_LV).addTo(sAssemblerRecipes);
```

Comment:
Use `SECONDS` (from `GT_RecipeBuilder`) for duration - makes recipes more readable.

Reply:
Thanks for the catch, they're all using this now.